### PR TITLE
Update data model for boats

### PR DESCRIPTION
**Pull request by igordavydsson**
In order to unify the measurements across different models,
we will ensure that width/length/draught are measured in meters
and stored as decimals, while weight is measured in kilos and 
stored as integers.
Changes to other data models will come later in separate PRs.

### CODE REVIEW

File: customers/models.py [hunk 1]
```diff
@@ -50,22 +53,27 @@ class Boat(TimeStampedModel, UUIDModel):
 
     # Dimensions
     length = models.DecimalField(
-        verbose_name=_("length"), decimal_places=2, max_digits=5
+        verbose_name=_("length (m)"),
+        decimal_places=2,
+        max_digits=5,
+        validators=[MinValueValidator(Decimal("0.01"))],
```
Comment (charn) [inline customers/models.py]: That is a very tiny ship. :joy:

File: customers/migrations/0004_change_boat_weight_to_int.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ("customers", "0003_add_validators_to_boat_dimensions"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="boat",
+            name="weight",
+            field=models.PositiveIntegerField(
```
Comment (charn) [inline customers/migrations/0004_change_boat_weight_to_int.py]: Will this automatically get rid possible decimals i.e. is it safe to do without a data migration or do we even need to care about the data at this point?

File: customers/models.py (hunk 1 above)
Comment (igordavydsson) [reply customers/models.py]: for paper boats 🙃 ⛵️

File: customers/migrations/0004_change_boat_weight_to_int.py (hunk 2 above)
Comment (igordavydsson) [reply customers/migrations/0004_change_boat_weight_to_int.py]: we do not have any data at this point, so I just skipped data migration.

**Pull request merged**